x/authz/keeper/keeper.go

```diff
@@ -256,3 +266,28 @@ func (k Keeper) InitGenesis(ctx sdk.Context, data *authz.GenesisState) {
 		}
 	}
 }
+
+// insertIntoGrantQueue Inserts a grant key into the grant queue
+func (keeper Keeper) insertIntoGrantQueue(ctx sdk.Context, grantKey []byte, expiration time.Time) {
+	store := ctx.KVStore(keeper.storeKey)
+	store.Set(GrantQueueKey(grantKey, expiration), grantKey)
```

Comment:
@robert-zaremba  We are duplicating `grantKey` in both key & value to prevent a collision.

Another solution is, using a slice for storing the `grantKey`.
```
message GrantQueueItem {
  repeated GGMTriple ggm_triples = 1; 
}

message GGMTriple {
  string granter = 1;
  string grantee = 2;
  string msg_type_url = 3;
}
```